db/config.cc

```diff
@@ -909,6 +912,11 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , task_ttl_seconds(this, "task_ttl_in_seconds", liveness::LiveUpdate, value_status::Used, 10, "Time for which information about finished task stays in memory.")
     , cache_index_pages(this, "cache_index_pages", liveness::LiveUpdate, value_status::Used, false,
         "Keep SSTable index pages in the global cache after a SSTable read. Expected to improve performance for workloads with big partitions, but may degrade performance for workloads with small partitions.")
+    , auth_superuser_name(this, "auth_superuser_name", value_status::Used, "", 
+        "Initial authentication super username. Ignored if authentication tables already contain a super user")
```

Comment:
Why not default to cassandra?

Reply:
No real reason other than a deep desire to move away from that default...

Reply:
I think it's okay to put it there. Removes a helper, and easier for the users to determine what the default it.

Reply:
My only argument against is that by this logic, the password option should have 'cassandra' as salted hash as default. However, this requires the deployment host to have said hash, which might require us to limit it to a less secure hash? (I would really suspect all expected platforms to have SHA512, but...). A very ugly default in any case... (Output of mkpasswd)